tests/keras/layers/convolutional_test.py

```diff
@@ -225,6 +225,25 @@ def test_conv2d_transpose(padding, out_padding, strides):
                fixed_batch_size=True)
 
 
+@keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason='cntk only supports dilated conv transpose on GPU')
+def test_conv2d_transpose_dilation():
```

Comment:
Is this unit test really sufficient?

Reply:
We can specify the input, a constant kernel and check the output, but due to Theano backend need to rotate the kernel which makes the test complicated, and we need to hard code output matrix. The following code is I run for offline test, I will wrap them as a unit test.
```
from keras.utils.conv_utils import convert_kernel
kernel = np.arange(27).reshape((3, 3, 1, 3))
print(kernel.shape)
# kernel = convert_kernel(kernel) # Enable this line for Theano backend

inputs = Input(shape=(5, 6, 3))
m = Conv2DTranspose(1, kernel_size=3, padding='same', dilation_rate=2, kernel_initializer=Constant(kernel))(inputs)
model = Model(inputs=inputs, outputs=m)
model.summary()

x = np.arange(180).reshape((2, 5, 6, 3)).astype(np.float32)
y = model.predict(x)

print(np.squeeze(y))
print(y.shape)
```